crates/biome_grit_formatter/src/grit/lists/definition_list.rs

```diff
@@ -5,6 +5,16 @@ pub(crate) struct FormatGritDefinitionList;
 impl FormatRule<GritDefinitionList> for FormatGritDefinitionList {
     type Context = GritFormatContext;
     fn fmt(&self, node: &GritDefinitionList, f: &mut GritFormatter) -> FormatResult<()> {
-        format_verbatim_node(node.syntax()).fmt(f)
+        let mut join = f.join_nodes_with_hardline();
+
+        for definition in node {
+            let def_clone = definition.clone().unwrap();
```

Comment:
Is `unwrap` safe here?

Reply:
Thanks for calling these out as well! I added these in earlier to help with debugging, but it's not necessary to clone here.

Reply:
Thanks for calling this out! I was following an example from the GraphQL formatter for [definition lists](https://github.com/branberry/biome/blob/a6593fa5e1e524bc2859fb6916cfd3404dba40c6/crates/biome_graphql_formatter/src/graphql/lists/definition_list.rs#L6), but I didn't fully understand why I was getting a `Result<AnyGritDefinition, SyntaxError>` back as opposed to just `AnyGritDefinition`, but I realize now it's due to [the GritDefinitionList implementing the AstSeparatedList trait](https://github.com/branberry/biome/blob/a6593fa5e1e524bc2859fb6916cfd3404dba40c6/crates/biome_grit_syntax/src/generated/nodes.rs#L12625) as opposed to [GraphQL implementing the AnyNodeList trait](https://github.com/branberry/biome/blob/a6593fa5e1e524bc2859fb6916cfd3404dba40c6/crates/biome_graphql_syntax/src/generated/nodes.rs#L7650). 

I can handle the the error here and not use `unwrap`, but I'm now wondering if the `AstSeparatedList` makes sense for the `AnyGritDefinition` struct. @arendjr, any thoughts on this?

Reply:
I’m not sure either, to be honest. I could imagine it has to do with the newlines that must be used as separate between Grit definitions?

Reply:
Hey Arend! After a bit more research, I believe it might have to do with the fact that definitions also include any pattern or predicate. One example is the `or` pattern, which separates arguments by a comma. I couldn't find an instance, though, where the definition list was greater than one. I could do the work of having a separator implementation, but it might be a bit out of scope for this PR as it would probably introduce a good amount of code.

I think for now, I'll just handle the error in a more rudimentary way until we have a better idea of when we might want the separator logic.

Reply:
Sounds good to me!